bin/bootstrax

```diff
@@ -659,7 +667,7 @@ def sufficient_diskspace():
             send_heartbeat(dict(state='disk full'))
     set_state(dead_state)
     message = f"No disk space to write to. Kill bootstrax on {hostname}"
-    log_warning(message, priority='warning')
+    log_warning(message, priority='Error')
```

Comment:
We should be consistent with capitalization of message levels. Also, this is something that redax would log with 'fatal' priority, because it's this process's last dying gasp before it dies.